demo/api/src/db/fixtures/generators/image-generator.service.ts

```diff
@@ -0,0 +1,34 @@
+import { download, File, FilesService } from "@comet/cms-api";
+import { Injectable } from "@nestjs/common";
+import faker from "faker";
+
+@Injectable()
+export class ImageGeneratorService {
+    constructor(private readonly filesService: FilesService) {}
+
+    public async generateImages(imageCount: number): Promise<File[]> {
+        const imageFiles = [];
```

Comment:
Wouldn't it be better if the ImageGeneratorService had the image files as a private member and provide a `getRandomImage()` function that chooses an image. Otherwise the imagesFiles need to be passed to all other generators

Reply:
good idea, there is a getRandomImage function now which will be used in future MR's in blocks directly